### ISSUE TITLE
Compatibility table

### ISSUE THREAD

**Issue by talonchandler** [mehta-lab/recOrder#362]
I would like to build a table of compatibility notes, either as its own file or as part of `microscope-installation-guide.md`. 

Here's an initial draft with entries I know about:

| recOrder version | 0.3.0  | 0.4.0 |
| :--- | :--- | :---|
| OS | Windows 10 | Windows 10 |
| Micromanager version | [2022-09-20 (160 MB)](https://download.micro-manager.org/nightly/2.0/Windows/MMSetup_64bit_2.0.1_20220920.exe) | [2023-04-26 (160 MB)](https://download.micro-manager.org/nightly/2.0/Windows/MMSetup_64bit_2.0.1_20230426.exe)  | 
| Meadowlark PC software version | 1.04 | 1.08 | 
| Meadowlark controller firmware version | >=1.04 | >=1.04 |
| Meadowlark drivers | [recOrder-0.3.0-device-drivers.zip (112 kB)](https://github.com/mehta-lab/recOrder/releases/download/0.3.0/recOrder-0.3.0-device-drivers.zip) | (link to usb device driver after github release, but before PyPI release) |

I've found the Meadowlark software version to be particularly critical because it is non-trivial to upgrade/downgrade. I found that installing recOrder onto an existing system with Meadowlark software version 1.08 forced my hand to install a (currently unreleased) 0.4.0dev version.

**Comment by mattersoflight** [comment]
Thanks, @talonchandler! As discussed offlin e, please turn this issue into a doc where we can document the dependency between the Meadowlark firmware -> device driver -> CellDrive -> our Micro-Manager device adapter.

**Comment by talonchandler** [comment]
@ieivanov and I just had a chat about how to handle this:

1. We decided that when we release `recOrder-napari==0.4.0` we will distribute the `usbdrvd.dll` built against V108 via the `recOrder` GitHub release page. 
2. We'll link to this driver and instruct the user to do drag-and-drop it into their MM installation. We'll put the instructions in the `recOrder` docs. It doesn't seem like we'll need to link to the micromanager device driver instruction page. 
3. We decided to keep installation instructions with their versions---no need to maintain legacy documentation. For example, 0.3.0 documentation will live with the 0.3.0 branch/release and 0.4.0 documentation will live with the 0.4.0 branch/release.

**Comment by talonchandler** [comment]
@ieivanov and I discussed this table as well, and we decided to keep these compatibility notes with each release. We don't think there's a need to maintain a complete table along with legacy installation instructions. 

- I'll release a 0.3.1 patch release with these non-automated dependencies added to the docs (just the 0.3. dependencies)
- I'll add the 0.4 non-automated dependencies to the docs, and these instructions will live with the 0.4 release.
